index.js

```diff
@@ -76,48 +78,72 @@ function showMyMeal() {
   var randomSide = sides[getRandomIndex(sides)];
   var randomMainDish = mainDish[getRandomIndex(mainDish)];
   var randomDessert = dessert[getRandomIndex(dessert)];
-  
+  // var entireMeal = randomSide + randomMainDish + randomDessert;
+
   if (sideRadioButton.checked === true) {
     getRandomSide(randomSide);
-  } 
+  }
   if (mainDishRadioButton.checked === true) {
-     getRandomMainDish(randomMainDish);
+     getRandomMainDish(randomMainDish); 
   } 
   if (dessertRadioButton.checked === true) {
-    getRandomDessert(randomDessert) 
+    getRandomDessert(randomDessert);
+  } 
+  if (entireMealRadioButton.checked === true) {
+    getEntireMeal(randomSide, randomMainDish, randomDessert);
+  } 
+  else {
+    showErrorMessage()
   }
+  showClearButton();
 };
 
 function getRandomSide(side) {
   rightBoxContent.innerHTML = `
   <h1 class="right-box-upper-message">You should make:<h1>
-  <h2 class="right-box-main-message">${side}</h2>
-  <button class="clear-button" type="button">CLEAR</button>`
+  <h2 class="right-box-main-message">${side}</h2>`
 };
 
 function getRandomMainDish(main) {
   rightBoxContent.innerHTML = `
     <h1 class="right-box-upper-message">You should make:<h1>
-    <h2 class="right-box-main-message">${main}</h2>
-    <button class="clear-button" type="button">CLEAR</button>`
+    <h2 class="right-box-main-message">${main}</h2>`
 };
 
 function getRandomDessert(dessert) {
   rightBoxContent.innerHTML = `
   <h1 class="right-box-upper-message">You should make:<h1>
-  <h2 class="right-box-main-message">${dessert}</h2>
-  <button class="clear-button" type="button">CLEAR</button>`
-  
```

Comment:
with showing the "clear" button here i wasnt able to make it function correctly.  I believe it has to do with bubbling and event delegation.  I removed it from each innerHTML.